_browser/blockstack_storage.md

```diff
@@ -220,48 +220,36 @@ These are the `UserSession.putFile`, `UserSession.getFile`, and `lookupProfile`
 
 1. Open the `src/components/Profile.js` file.
 
-2. Replace the initial state in the `constructor()` method so that it holds the key properties required by the app.
+2. Replace the initial state in the component method so that it holds the key properties required by the app.
 
-    This code constructs a Blockstack `Person` object to hold the profile. Your constructor should look like this:
+    This code constructs a Blockstack `Person` object to hold the profile. Your component should look like this:
 
     ```javascript
-    constructor(props) {
-      super(props);
-
-      this.state = {
-        person: {
-          name() {
-            return 'Anonymous';
-          },
-          avatarUrl() {
-            return avatarFallbackImage;
-          },
-        },
-        username: "",
-        newStatus: "",
-        statuses: [],
-        statusIndex: 0,
-        isLoading: false
-      };
+    export const Profile = ({ userData, handleSignOut }) => {
```

Comment:
This is confusing because we're basically asking to replace the file I guess? It threw me off, especially because the component declaration is different in the file (`export default class Profile extends Component`) - no default, no extension

Reply:
I'm also left with a `componentWillMount` method that is now marked as unreachable (on the bottom of the page)